test/e2e/specs/wp-signup-spec.js

```diff
@@ -1536,23 +1399,6 @@ describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function() {
 			}
 		);
 
-		step( 'Can see the "Site Type" page, and enter some site information', async function() {
```

Comment:
Just noting that you can also hide the steps with `xstep()`, if you don't want to remove the tests since the code isn't removed (just hidden) either.